plugins/kubernetes/test/models/watchers/deploy_watcher_test.rb

```diff
@@ -9,6 +9,9 @@
   let(:project) { current_release.project }
 
   before do
+    # Disable multithreading so celluloid stays in the same transaction
+    ActiveRecord::Base.stubs(connection: ActiveRecord::Base.connection)
```

Comment:
found that on a very reliable blog ... so it must be good ;)
http://grosser.it/2016/04/06/testing-suckerpunchcelluloid-vs-activerecord-transactions/